js/templates/mobile-nav.hbs

```diff
@@ -11,7 +11,6 @@
         <label class="t-sans" for="menu-search">Search for candidates and committees</label>
         <div class="combo--search--mini">
           <input id="menu-search" name="search" class="combo__input js-menu-search" type="text"
-          placeholder="Search for candidates"
```

Comment:
I know why we're doing this, but want to nudge that this may make a confusing interaction for folks since you cannot search across candidates AND committees at the same time yet: the search isn't integrated. In this case, the placeholder text reinforced you were searching just for candidates. Do you still want to go ahead @emileighoutlaw ?